### PR TITLE
fixed bug on load

### PR DESCRIPTION
**Pull request by imariel2d**
Fixed bug where token session wasn't being saved on first load

### CODE REVIEW

File: webapp/src/components/Main/Main.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import './style/style.scss';
 const Main = () => {
   const { path } = useParams();
   const darkThemeActive = useSelector((state) => state.player.darkTheme);
-  const [triggerUpdate, setTriggerUpdate] = useState();
+  const [ , setTriggerUpdate] = useState();
```
Comment (jmsalcido) [inline webapp/src/components/Main/Main.jsx:13]: this can be

``` javascript
const [ setTriggerUpdate ] = useState();
```

**Review by jmsalcido — COMMENTED** [review]
is there a test for this?, good idea would be to have a test to verify that the `token` is set when this action is called.

at least a gif/video/loom for reference.

**Comment by imariel2d** [comment]
https://www.loom.com/share/a3b3930456d443219f8b94520a60e090

**Review by jmsalcido — APPROVED** [review]
LGTM - Looks Good To Me - Bien hecho!

**Pull request merged**